### DISCUSSION TITLE
Printing API

### DISCUSSION THREAD

**Discussion by veldfolds** [CommunityToolkit/Maui#1574] [New Feature Discussions]
We need a printing API in MAUI

**Comment by bijington** [comment]
Do you have any thoughts or proposals on how it should look or work?

**Reply by veldfolds** [reply]
First thanks for getting back to me, perhaps more like the API'S  in the 'Microsoft.Maui.ApplicationModel' namespace.
A simple example:

// Gets a  list of printers & returns null if no printer is attached
var printers = await Printing.Default.GetPrinters();

// Check if printing is supported in the attached device
Printer printer = printers.First();

// Check if printer is online and can print
bool isPrintingSupported = Printing.Default.IsPrintingSupported(printer);

// print a document
bool printResult = Printing.Print(myFile);

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers. 

If you feel this is being closed in error please let us know and we can re-open.
